docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-sphinx
+sphinx<=1.5.6
```

Comment:
why `<= 1.5.6` ?

Reply:
Because >1.5.6 it doesn't generate html: https://github.com/sphinx-doc/sphinx/issues/3800

Reply:
oh, I see.

Reply:
I belive, after converting all things to `rst` we will be able to remove this restriction and uncomment **html5** test (Sphinx 1.6 is expected to support htm5 [writer](https://github.com/sphinx-doc/sphinx/pull/3464)) because at present it generates dozens of errors and warnings.